Slash/Apache/User/User.pm

```diff
@@ -451,7 +451,7 @@ sub handler {
 	# These next two lines set a CSP for users, unless they are an admin who needs inline scripts for the admin bells and whistles
 	# This next line is correct. Changing it temporarily until we can get all the resources coming from the domain requested instead of the base domain
 	#if(!$user->{is_admin}) { $r->headers_out->set('Content-Security-Policy', "default-src 'none'; script-src https://checkout.stripe.com; frame-src https://checkout.stripe.com; connect-src https://checkout.stripe.com; img-src 'self' https://www.paypalobjects.com https://q.stripe.com; style-src 'unsafe-inline' 'self' https://checkout.stripe.com");}
-	if(!$user->{is_admin}) { $r->headers_out->set('Content-Security-Policy', "default-src 'none'; script-src https://checkout.stripe.com; frame-src https://checkout.stripe.com; connect-src https://checkout.stripe.com; img-src 'self' https://soylentnews.org https://www.soylentnews.org http://7rmath4ro2of2a42.onion https://www.paypalobjects.com https://q.stripe.com; style-src 'unsafe-inline' 'self' https://soylentnews.org https://www.soylentnews.org http://7rmath4ro2of2a42.onion https://checkout.stripe.com");}
+	if(!$user->{is_admin}) { $r->headers_out->set('Content-Security-Policy', "default-src 'none'; script-src 'self' https://soylentnews.org https://www.soylentnews.org http://7rmath4ro2of2a42.onion https://checkout.stripe.com; frame-src https://checkout.stripe.com; connect-src https://checkout.stripe.com; img-src 'self' https://soylentnews.org https://www.soylentnews.org http://7rmath4ro2of2a42.onion https://www.paypalobjects.com https://q.stripe.com; style-src 'unsafe-inline' 'self' https://soylentnews.org https://www.soylentnews.org http://7rmath4ro2of2a42.onion https://checkout.stripe.com");}
```

Comment:
'self' makes it work on dev or whatever the server thinks its hostname should be. I went ahead and explicitly spelled out the www and onion hosts as well as putting the base domain in too so we didn't have to make sure all our assets were done correctly. I thought that would be a good idea since I know at least some of them are not but I don't know offhand which they are.